src/Listener/ConsoleListener.php

```diff
@@ -16,14 +16,14 @@
 class ConsoleListener
 {
     /** @var EventDispatcherInterface */
-    protected $eventDispatcher = null;
+    protected $eventDispatcher;
 
     /**
      * Time when command started
      *
      * @var float
      */
-    protected $startTime = null;
+    protected $startTime;
```

Comment:
If for some reason onException or onTerminate are called before onCommand we will get an exception (L.76) because startTime will not have been set. We should check if startTime is set before dispatching.

Reply:
I am struggling with the cs-fixer. I initialized these vars inside the constructor